LBHackney-IT/lbh-social-care
# align workers api

**What**
- aligned workers api response
- removed superfluous allocatedBy

**How to test it**
- check `http://dev.hackney.gov.uk:3000/people/7/allocations/add?teamId=24`
- workers should be visible

**Note**
Allocations are broken as we need to add test users as workers
